src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -0,0 +1,59 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEngineering Wikia
+ */
+public class TestAdsTrackingPixels extends TemplateNoFirstLoad {
+
+  public static final ImmutableList<String> TRACKING_PIXELS_URL =
+      new ImmutableList.Builder<String>()
+          .add("http://beacon.krxd.net/pixel.gif")
+          .add("http://b.scorecardresearch.com/b")
+          .add("http://pixel.quantserve.com")
+          .build();
+
+  public static final ImmutableList<String> LINK_NAMES =
+      new ImmutableList.Builder<String>()
+          .add("Article1")
+          .add("Article2")
+          .add("Article3")
+          .add("Article2")
+          .add("Article1")
+          .add("Wikia Ad Testing")
+          .add("Article1")
```

Comment:
The test is synthetic, so I prepared these articles for testing functionality on consecutive page views.

One thing that the last "Article1" is not needed in the last version of the test.
(because I moved main assertion (previously I did assertion before click on link, right now - after click on link))

The logic behind these links and their order is simple:
1. I wanted to have several page views (more than just one page view on the main page and one page view on the article page)
2. We started from the Main page and I wanted to return to point where test started.

In this case I think the test truly checks the tracking pixels on the consecutive page views.
If you have you thoughts about order and articles feel free to share. :cat: